assets/out

```diff
@@ -40,47 +40,59 @@ source_org=$(jq -r '.source.org //empty' < $payload)
 source_space=$(jq -r '.source.space //empty' < $payload)
 params=$(jq -r '.params //empty' < $payload)
 
+cf_config_path=$(jq -r '.params.cf_config_path //empty' < $payload)
+
 cf_color=$(jq -r '.source.cf_color //empty' < $payload) && [ -n "$cf_color" ] && export CF_COLOR=$cf_color
 cf_dial_timeout=$(jq -r '.source.cf_dial_timeout //empty' < $payload) && [ -n "$cf_dial_timeout" ] && export CF_DIAL_TIMEOUT=$cf_dial_timeout
 cf_trace=$(jq -r '.source.cf_trace //empty' < $payload) && echo "$cf_trace" | grep -qi 'true' && export CF_TRACE=true
 command_file=$(echo "$params" | jq -r '.command_file //empty')
 
-if [ -z "$api" ]; then
-  printf '\e[91m[ERROR]\e[0m invalid payload (missing api)\n'
-  exit 1
-fi
 
-if [ -n "$username" ] && [ -n "$client_id" ]; then
-  printf '\e[91m[ERROR]\e[0m invalid payload (must specify only username or client_id)\n'
-  exit 1
-fi
+if [ -n "$cf_config_path" ]; then
+    cp "$cf_config_path" ~/.cf/config.json
```

Comment:
Instead of specifying the `cp_config_path` and then copying the file to the `~/.cf` folder, we could simplify this by allowing the user to specify an alternative `CF_HOME` environment variable that points to a folder that contains the new `config.json`.

Something like:
```
cf_home=$(jq -r '.params.cf_home //empty' < $payload)
...
[ -n "$cf_home" ] && export CF_HOME=$cf_home
```
thoughts?

Reply:
See https://docs.cloudfoundry.org/cf-cli/getting-started.html#login for `CF_HOME` usage

Reply:
✅

Reply:
cf_plugin_home needs to stay $HOME because of the autopilot plugin